resources/chat-view/chat.js

```
@@ -0,0 +1,460 @@
+  });
+
+  modelDropdown.addEventListener('change', event => {
+    const selectedModel = event.target.value;
+    vscode.postMessage({ command: 'updateModel', model: selectedModel });
+  });
+
+  window.addEventListener('message', event => {
+    const message = event.data;
+    switch (message.command) {
+      case 'updateChatLog':
+        const sanitizeHtml = DOMPurify.sanitize(message.chatLogHtml);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)